### ISSUE TITLE
Error message appears every time I open it

### ISSUE THREAD

**Issue by hw2007** [2009-Nissan-Cube/About-This-Hack#112]
Every time I open the app, this message appears.
<img width="262" alt="image" src="https://github.com/user-attachments/assets/634e1bc8-1a0e-4c56-9a8f-3e76a2712afc">

Clicking "Return" allows me to continue using the app as normal, but the message reappears next time I open it.

**Comment by 0xCUB3** [comment]
Yeah this is likely because I switched the repo to my enterprise account. This will be fixed in the next release. Thanks for submitting 👍🏻

**Comment by hw2007** [comment]
> Yeah this is likely because I switched the repo to my enterprise account. This will be fixed in the next release. Thanks for submitting 👍🏻

Wow fastest reply I've ever seen! Thanks!
